src/components/LabelPicker/LabelPicker.scss

```diff
@@ -0,0 +1,39 @@
+.LabelPicker {
+    &__dialog {
+        z-index: 400;
+    }
+
+    &__container {
+        background-color: var(--color-background);
+        border: 1px solid var(--color-neutral);
+        border-radius: var(--border-radius);
+        box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.1);
+        padding: var(--spacing-2x);
+    }
```

Comment:
I think we need to add `max-width` for the popup, otherwise, the long label stretches the container or maybe we can allow the application itself to control block width via props/styles/classes. Also nice to have truncating for the long labels `...`
![Screenshot 2022-04-26 at 12 15 20](https://user-images.githubusercontent.com/19436304/165279742-866d350c-b236-4ac3-9310-fc6d72e576c5.png)